docs/azure/UnwiredLabsCellGeolocation.rst

```diff
@@ -5,25 +5,32 @@ Unwired Labs Cell Geolocation
 
 You can enable the :ref:`cell geolocation lookup <app-cellgeolocation>` for single cells for the application using `Unwired Labs' <https://unwiredlabs.com/>`_ geolocation API in your deployment.
 
-To use Unwired Labs' LocationAPI, provide the ``unwiredlabsApiKey`` parameter when deploying the solution:
+To use Unwired Labs' LocationAPI, set the ``enableUnwiredLabsCellLocation`` parameter to ``true`` when deploying the solution:
 
-.. parsed-literal::
-   :class: highlight
+.. code-block:: bash
 
-    az deployment group create \\
-        --resource-group ${RESOURCE_GROUP:-nrfassettracker} \\
-        --mode Complete \\
-        --name enable-unwiredlabs \\
+    az deployment group create \
+        --resource-group ${RESOURCE_GROUP:-nrfassettracker} \
+        --mode Complete \
+        --name enable-unwiredlabs \
         --template-file azuredeploy.json \
-        --parameters \\
-            appName=${APP_NAME:-nrfassettracker} \\
-            appRegistrationClientId=$APP_REG_CLIENT_ID \\
-            b2cTenant=${B2C_TENANT:-nrfassettrackerusers} \\
-            unwiredlabsApiKey=*your API key*
+        --parameters \
+            appName=${APP_NAME:-nrfassettracker} \
+            appRegistrationClientId=$APP_REG_CLIENT_ID \
+            b2cTenant=${B2C_TENANT:-nrfassettrackerusers} \
+            keyVaultName=${APP_NAME:-nrfassettracker} \
+            enableUnwiredLabsCellLocation=true
 
 This command enables the `geolocateCellFromUnwiredLabs` function to resolve cells.
```

Comment:
Should we have the function name between double back ticks to display it as `geolocateCellFromUnwiredLabs`?

Reply:
Right, this is markdown, not restructuredtext.
```suggestion
This command enables the ``geolocateCellFromUnwiredLabs`` function to resolve cells.
```